<!DOCTYPE html>
<html>
<head>
  <title>normjam: A Normalization Workshop and Idea Jamboree</title>
  <link rel="stylesheet" type="text/css" href="stylesheets/main.css">
  <link rel="stylesheet" type="text/css" href="stylesheets/footer.css">
  <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
</head>
<body>

  <nav class="navigation">
    <ul>
      <li><a href="index.html">Home</a></li>
      <li><a href="resources.html">Resources</a></li>
      <li><a href="agenda.html">Agenda</a></li>
      <li><a href="attendees.html">Attendees</a></li>
    </ul>
  </nav>

  <div id="banner">
    <div class="content">
      <h1>normjam</h1>
    </div>
  </div>

  <div id="main" class="content">
    <h2>a normalization workshop and jamboree</h2>
    
    <h3>when and where</h3>
    
    <p>NormJam is a 2 day workshop on the topic of normalization of single cell data hosted by the <a href="https://chanzuckerberg.com/">Chan Zuckerberg Initiative</a> and the <a href="https://www.nygenome.org/">New York Genome Center</a>.</p>
     <p>The workshop will be held on November 19th and 20th, 2019 at the New York Genome Center, 101 6th Ave, New York, NY 10013.
</p>
    
    <h3>about normjam</h3>
   
    <p>Normalization refers to the pre-processing of a single expression matrix from a scRNA-seq dataset, in isolate, to remove technical variation in read counts across cells and molecules/genes, while maintaining biological variability. 
</p>
   
    <h4>the problem</h4>
    
    <p>There are a wide variety of algorithms that exist for normalization, aka within sample removal of technical variability. Each of these algorithms result in differing outputs, and any error from normalization propagates to all subsequent analysis steps. So it is really important to normalize properly!</p>
    <p>However, it is hard to evaluate which normalization algorithm to use and when. Many computational biologists working in single cell biology have been focused on downstream analysis, like batch effect correction (across sample removal of technical variability), data integration (within a modality, across experiments, assays, etc.), and multi-modal data integration (e.g., integrating smFISH with scRNA-seq data). We want to spend a few days thinking deeply about this problem and doing some initial benchmarking of methods.</p>
    
    <h4>our goals</h4>
    <ul>
      <li>Bring together experts working on normalization to discuss and ideate</li>
      <li>Define normalization, together</li>
      <li>Apply existing normalization algorithms to a common set of benchmark data from different tissues and technologies to compare and contrast performance</li>
      <li>Communicate our findings to the broader community</li>
    </ul>
  
    <h4>our questions</h4>
    <ul>
      <li>How do we precisely define normalization?</li>
      <li>How do we define what success looks like for a normalization algorithm? What metrics should we use?</li>
      <li>What data should we use for benchmarking? How many tissues? How many modalities?</li>
    </ul>
	  
	  <footer class="footer-distributed">
	    <div class="footer-right">
		    <a href="#"><i class="fa fa-facebook"></i></a>
		    <a href="#"><i class="fa fa-twitter"></i></a>
		    <a href="#"><i class="fa fa-linkedin"></i></a>
		    <a href="#"><i class="fa fa-github"></i></a>
	    </div>
	    
	    <div class="footer-left">
		    <p class="footer-links">
			    <a href="#">Home</a>
			    <a href="#">Resources</a>
			    <a href="#">Agenda</a>
			    <a href="#">Attendees</a>
			    <a href="#">Contact</a>
		    </p>
	    </div>
	  </footer>
    
</body>
</html>
